_artifacts/opensearch-1.0/all/opensearch-1.0.1-freebsd-all.markdown

```diff
@@ -0,0 +1,12 @@
+---
+role: daemon
+artifact_id: opensearch
+version: 1.0.1
+platform: freebsd
+architecture: all
+slug: opensearch-1.0.1-freebsd-all
+category: opensearch
+type: pkg
```

Comment:
Humm. `pkg` is a bit overloaded as it's used in macos too.

Reply:
Quite agree… Maybe something like 'system-package' to insist on the way this is managed using the regular package manager of the system?